frontend/src/action-creator.sync.js

```diff
@@ -113,14 +116,20 @@ export const updateWorkspaceDetail = workspaceDetail => ({ type: `${UPDATE}/${WO
 export const WORKSPACE_LOADED = `${WORKSPACE}/Loaded`
 export const setWorkspaceLoaded = () => ({ type: `${SET}/${WORKSPACE_LOADED}` })
 
-export const WORKSPACE_MEMBER = `${WORKSPACE}/Member`
+const MEMBER = 'Member'
+export const WORKSPACE_MEMBER = `${WORKSPACE}/${MEMBER}`
 export const WORKSPACE_MEMBER_LIST = `${WORKSPACE_MEMBER}/List`
 export const setWorkspaceMemberList = workspaceMemberList => ({ type: `${SET}/${WORKSPACE_MEMBER_LIST}`, workspaceMemberList })
+// INFO - CH - 2023-11-22 - WORKSPACE_MEMBER_ADD is used by api function postWorkspaceMember()
+// but nothing in bind to it
 export const WORKSPACE_MEMBER_ADD = `${WORKSPACE_MEMBER}/${ADD}`
+// INFO - CH - 2023-11-22 - WORKSPACE_MEMBER_ADD is used by api function deleteWorkspaceMember()
+// but nothing in bind to it
```

Comment:
> but nothing in bind to it

wouldn't it be "but nothing is bound to it" ?

Reply:
will fix